test/soca/gw/CMakeLists.txt

```diff
@@ -10,25 +10,14 @@ add_test(NAME test_gdasapp_soca_prep
       ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/ush:${PROJECT_SOURCE_DIR}/../../ush/python/wxflow/src:$ENV{PYTHONPATH}")
 
 # Identify machine
-set(MACHINE "container")
-IF (IS_DIRECTORY /work2)
-  IF (IS_DIRECTORY /apps/other)
-    set(MACHINE "hercules")
-    set(PARTITION "hercules")
-  ELSE()
-    set(MACHINE "orion")
-    set(PARTITION "orion")
-  ENDIF()
-ENDIF()
-IF (IS_DIRECTORY /scratch2/NCEPDEV/)
-  set(MACHINE "hera")
+if (MACHINE STREQUAL "hercules")
```

Comment:
Ha! I was going to work on this today ... Thanks for the fix @RussTreadon-NOAA !

Reply:
@guillaumevernieres , if you are OK with the changes in this PR I can merge pending a review & approval

Reply:
Yes, looks good @RussTreadon-NOAA . Thanks again.